envision/web/src/index.js

```diff
@@ -29,7 +29,7 @@ import { ToastProvider } from "react-toast-notifications";
 import Snack from "./components/snack";
 
 let client = new Client({
-  endpoint: window.location.origin,
+  endpoint: "http://localhost:8081",
```

Comment:
This was changed in https://github.com/huawei-noah/SMARTS/commit/eeea1d6c6ba4a93e70e110385a5cfb12bead4bea

But I am getting the following error when I run `npm start`: 
`Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0` with `window.location.origin=http://localhost:8080`

So I reverted back the change.

Reply:
This is where `window.location.origin` was introduced https://github.com/huawei-noah/SMARTS/pull/236/files#r533447521

Reply:
When you made some changes and re-compiled the distribution using `npm run build`, does the new build include your changes? For me it still does not with `window.location.origin`, even when I use the `scl` command

Reply:
The new build did reflect my changes. Maybe something went wrong during `npm run build`.

Reply:
Ok I cleared all browser cache and it works now, sorry for the confusion.